ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/useDeployments.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from '@apollo/client';
+import type { QueryHookOptions } from '@apollo/client';
+
+import { getPaginationParams } from 'utils/searchUtils';
+import type { ApiSortOption } from 'types/search';
+import type useURLPagination from 'hooks/useURLPagination';
+import { deploymentListQuery } from '../Tables/DeploymentOverviewTable';
+import type { Deployment } from '../Tables/DeploymentOverviewTable';
+
+export function useDeployments({
```

Comment:
If you did not yet discuss benefits of **named** instead of **default** export (for **other than** individual React components, I am guessing) with the team, can you put on agenda for next Office Hours?